Theme.ts

```diff
@@ -128,7 +125,7 @@ export const GlobalStyle = createGlobalStyle`
   }
 
   h1, h2, h3, h4, h5, h6 {
-    color: ${(props) => props.theme.colours.teal};
+    color: ${(props) => props.theme.colours.plum};
```

Comment:
Plum isn't a brand colour for The Point :-/

Reply:
Yeah - sorry they like it. I think also they want a bit of darts in there which is why they like the pink too. We've had lots of conversations about the guidelines and they are happy withh this.